sway-lib-std/src/hash.sw

```diff
@@ -137,23 +139,35 @@ impl Hash for Bytes {
     }
 }
 
-impl<A, B> Hash for (A, B) where A: Hash, B: Hash  {
-    #![inline(never)]
+impl<A, B> Hash for (A, B) where
```

Comment:
`where` clauses by default should format to be on the next line, just like how they are with functions. This should be another issue, easy fix too.

Reply:
I think in this case, if the where body is too long `where` should be in the next line. I'll improve it.

Reply:
There is an option `SameLineWhere` in the config settings, and that is the only time `where` should be on the same line. The default config should result in the same kind of formatting as functions, structs and all other types that can have `where` clauses.